doc/CPP.md

```
@@ -30,6 +30,19 @@
 - [Array in C-major order](../examples/line_sender_cpp_example_array_c_major.cpp)
 - [Custom array type integration](../examples/line_sender_cpp_example_array_custom.cpp)
 
+**Decimal**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)